modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -0,0 +1,70 @@
+:experimental:
+= Migrating from CoreOS Container Linux (CL) to Fedora CoreOS (FCOS)
+
+Fedora CoreOS is the official successor of CoreOS Container Linux. This page attempts to document the differences between CL and FCOS to ease the transition to FCOS.
+
+:toc:
+
+== Introduction
+
+To migrate from CL to FCOS, you must convert your old Container Linux Configs, Ignition configs, or `cloud-config` files to a xref:fcct-config.adoc[Fedora CoreOS Config (FCC)] and adapt the contents for FCOS. Since many of the configuration details have changed, you should reference this page and the https://github.com/coreos/fedora-coreos-tracker/issues/159[CL migration issue] on GitHub.
+
+== Installation changes
+
+The following changes have been made to the installation process:
+
+* The `coreos-install` script has been replaced with https://github.com/coreos/coreos-installer[`coreos-installer`]. It offers similar functionality.
+* The `coreos.autologin` kernel command-line parameter is not currently supported in FCOS.
+* Certain CL platforms, such as Vagrant, are not yet supported in FCOS. Refer to the https://getfedora.org/coreos/download/[Download page] to see the available image types.
+
+== Software package changes
+
+* `etcd` is not included in FCOS. Refer to xref:running-containers.adoc[Running etcd] for instructions to run it as a container on FCOS.
```

Comment:
Not sure if it's possible to also link to an anchor but there's a `#running-etcd` anchor in that doc that might be a better link.